plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -13,33 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { Entity } from '@backstage/catalog-model';
 import { Table, TableColumn } from '@backstage/core';
 import { Link } from '@material-ui/core';
 import { Alert } from '@material-ui/lab';
 import React, { FC } from 'react';
 import { generatePath, Link as RouterLink } from 'react-router-dom';
-import { Component } from '../../data/component';
 import { entityRoute } from '../../routes';
 
 const columns: TableColumn[] = [
   {
     title: 'Name',
-    field: 'name',
+    field: 'metadata.name',
     highlight: true,
-    render: (componentData: any) => (
+    render: (entity: any) => (
```

Comment:
Can be now `entity:Entity`, right?

Reply:
unfortunately not according to the signature of that interface :/